package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mdjson-schemas",
-  "version": "2.8.0-rc5",
+  "version": "2.8.0-rc6",
```

Comment:
@timothypage It looks like there was an rc8 that is 2.8.0 in RubyGems. There are differences between the 2 versions. Should those differences get pulled in?

https://rubygems.org/gems/adiwg-mdjson_schemas/versions/2.8.0